core/src/main/java/org/infinispan/interceptors/distribution/TriangleDistributionInterceptor.java

```diff
@@ -401,6 +405,7 @@ private void sendToBackups(DistributionInfo distributionInfo, DataWriteCommand c
       if (trace) {
          log.tracef("Command %s got sequence %s for segment %s", id, sequenceNumber, segmentId);
       }
+      // TODO Should we use sendToAll in replicated mode?
```

Comment:
there is no triangle in replicated mode :)

Reply:
Yes, but there could be :)